src/transformers/models/flaubert/tokenization_flaubert.py

```diff
@@ -247,6 +247,7 @@ def __init__(
         self.cache = {}
 
         super().__init__(
+            do_lowercase=self.do_lowercase,
```

Comment:
not that it matters, but maybe keep `do_lowercase=do_lowercase` for consistency with the other params?